src/lib/button/button.scss

```diff
@@ -47,6 +47,12 @@
   }
 }
 
+[md-button], [md-raised-button] {
```

Comment:
Could you add a comment that explains what this style is for / why it was added?